### ISSUE TITLE
Slack Real Time Client bug for fileUpload

### ISSUE THREAD

**Issue by jcarinus** [mpociot/botman#262]
I've recently tested out this repo to write a new slack bot:

`<?php
require 'vendor/autoload.php';

use Mpociot\BotMan\Messages\Message;
use Mpociot\BotMan\BotManFactory;
use React\EventLoop\Factory;

$loop = Factory::create();
$botman = BotManFactory::createForRTM([
    'slack_token' => 'MY BOT TOKEN'
], $loop);

$botman->hears('test', function(Mpociot\BotMan\BotMan $bot) {
  $message = Message::create('Sample text')
              ->filePath('test.pdf');
  $bot->reply($message);
});

$loop->run();
?>`

Running this code produces the following error:

PHP Fatal error:  Uncaught Error: Call to undefined method Slack\RealTimeClient::fileUpload() in PROJECT_ROOT/vendor/mpociot/botman/src/Mpociot/BotMan/Drivers/SlackRTMDriver.php:142

Am I at fault here or is it an actual bug in the code?

**Comment by mpociot** [comment]
Hey,

are you sure your composer dependencies are up to date? 

I just built a RTM API bot using file uploads today.
Please try a `composer update` and see if that fixes the issue.

**Comment by jcarinus** [comment]
Thanks mpociot. I'll see and get back to you.

**Comment by jcarinus** [comment]
The update unfortunately didn't fix the issue.

**Comment by jcarinus** [comment]
I've found the problem. Sorry for wasting your time.

There was a composer.json file in my folder with the following contents:

`{
    "minimum-stability": "dev",
    "prefer-stable": true
}`

This was left over from when I initially tried the [https://github.com/sagebind/slack-client](https://github.com/sagebind/slack-client) repo. I did delete my vendor folder before trying out BotMan, but I forgot about the json file. Deleting the json file and the vendor folder and reinstalling BotMan fixed the issue for me.

Thank you for maintaining the repo @mpociot. You are possibly saving me a mountain of work.
